pkg/util/azureclient/authz/remotepdp/const.go

```diff
@@ -17,4 +17,11 @@ const (
 	Allowed    AccessDecision = "Allowed"
 	NotAllowed AccessDecision = "NotAllowed"
 	Denied     AccessDecision = "Denied"
+
+	// GroupExpansion is the value to be used with ClaimName in SubjectAttributes
+	// This value gives CheckAccess a hint that it needs to retrieve all the groups the principal belongs to
+	// and then give the response based on all group entitlements.
+	//
+	// https://eng.ms/docs/microsoft-security/identity/auth/access-control-managed-identityacmi/azure-authz-data-plane/authz-dataplane-partner-wiki/remotepdp/checkaccess/samples/requestresponse
+	GroupExpansion = `{"groups":"src1"}`
```

Comment:
@nwnt  Also, thanks a lot for the pointers to the docs as sometimes that kind of info is not easy to find. 🙂